odl/operator/__init__.py

```diff
@@ -34,3 +34,5 @@
 __all__ += pspace_ops.__all__
 
 from . import oputils
```

Comment:
My remark was actually more technical. If you do `from .oputils import *`, then `oputils` is also imported to the namespace, so the subsequent line makes sense.